pandas/_libs/parsers.pyx

```diff
@@ -1406,59 +1406,6 @@ cdef inline StringPath _string_path(char *encoding):
 # Type conversions / inference support code
 
 
-cdef _string_box_factorize(parser_t *parser, int64_t col,
```

Comment:
this is completely unused?

Reply:
Yea this is the only thing that appears in a grep of _libs

```python
$ grep -nr "_string_box_factorize" pandas/_libs
pandas/_libs/parsers.pyx:1409:cdef _string_box_factorize(parser_t *parser, int64_t col,
```

`cdef` as well so can't be called from outside of there